go/libraries/doltcore/sqle/binlogreplication/binlog_replication_test.go

```diff
@@ -623,9 +626,11 @@ func startMySqlServer(dir string) (int, *os.Process, error) {
 	primaryDatabase = sqlx.MustOpen("mysql", dsn)
 
 	os.Chdir(originalCwd)
-
 	fmt.Printf("MySQL server started on port %v \n", mySqlPort)
 
+	primaryDatabase.MustExec("CREATE USER 'replicator'@'%' IDENTIFIED BY 'password';")
+	primaryDatabase.MustExec("GRANT ALL PRIVILEGES ON *.* TO 'replicator'@'%';")
```

Comment:
Is this grant too broad?

Reply:
Yup – switched to `GRANT REPLICATION SLAVE`